examples/python/graphs/graphs.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+"""Examples of logging graph data to Rerun and performing force-based layouts."""
+
+from __future__ import annotations
+
+import argparse
+import itertools
+import random
+
+import numpy as np
+import rerun as rr
+import rerun.blueprint as rrb
+from rerun.blueprint.archetypes.force_collision_radius import ForceCollisionRadius
+from rerun.blueprint.archetypes.force_link import ForceLink
+from rerun.blueprint.archetypes.force_many_body import ForceManyBody
+from rerun.components.color import Color
+from rerun.components.radius import Radius
+from rerun.components.show_labels import ShowLabels
+
+color_scheme = [
+    Color([228, 26, 28]),  # Red
+    Color([55, 126, 184]),  # Blue
+    Color([77, 175, 74]),  # Green
+    Color([152, 78, 163]),  # Purple
+    Color([255, 127, 0]),  # Orange
+    Color([255, 255, 51]),  # Yellow
+    Color([166, 86, 40]),  # Brown
+    Color([247, 129, 191]),  # Pink
+    Color([153, 153, 153]),  # Gray
+]
+
+DESCRIPTION = """
+# Graphs
+This example shows various graph visualizations that you can create using Rerun.
+In this example, the node positions—and therefore the graph layout—are computed by Rerun internally using a force-based layout algorithm.
+
+You can modify how these graphs look by changing the parameters of the force-based layout algorithm in the selection panel.
+
+The full source code for this example is available
+[on GitHub](https://github.com/rerun-io/rerun/blob/latest/examples/python/graphs?speculative-link).
+""".strip()
+
+
+# We want reproducible results
+random.seed(42)
+
+
+def log_lattice(num_nodes) -> None:
+    coordinates = itertools.product(range(num_nodes), range(num_nodes))
+
+    nodes, colors = zip(*[
+        (
+            str(i),
+            rr.components.Color([round((x / (num_nodes - 1)) * 255), round((y / (num_nodes - 1)) * 255), 0]),
+        )
+        for i, (x, y) in enumerate(coordinates)
+    ])
+
+    rr.log(
+        "lattice",
+        rr.GraphNodes(
+            nodes,
+            colors=colors,
+            labels=[f"({x}, {y})" for x, y in itertools.product(range(num_nodes), range(num_nodes))],
+        ),
+        static=True,
+    )
+
+    edges = []
+    for x, y in itertools.product(range(num_nodes), range(num_nodes)):
+        if y > 0:
+            source = (y - 1) * num_nodes + x
+            target = y * num_nodes + x
+            edges.append((str(source), str(target)))
+        if x > 0:
+            source = y * num_nodes + (x - 1)
+            target = y * num_nodes + x
+            edges.append((str(source), str(target)))
+
+    rr.log("/lattice", rr.GraphEdges(edges, graph_type="directed"), static=True)
```

Comment:
either always use leading slash or never, otherwise readers get confused
```suggestion
    rr.log("lattice", rr.GraphEdges(edges, graph_type="directed"), static=True)